mkdocs.yml

```
@@ -40,6 +40,7 @@ nav:
     - Routing:
       - Route configuration: docs/routing/route_configuration_options.md
       - Match & cascade merge: docs/routing/route_match_patterns.md
+      - i18n & trailing slash: docs/routing/localisation_and_trailing_slash.md
```

```suggestion
      - i18n & trailing slash: docs/routing/localization_and_trailing_slash.md
```

Please rename the markdown file accordingly.